Readme.md

```diff
@@ -1,77 +1,130 @@
 # Capi
 
-> Capi is a work in progress. The documentation may not reflect the current implementation. **Expect a stable release and proper documentation in early 2023**.
+> Capi is a work in progress. The documentation may not reflect the current
+> implementation. **Expect a stable release and proper documentation in early
+> 2023**.
 
-Capi is a declarative, TypeScript-first toolkit for crafting interactions with Substrate-based chains. It consists of [FRAME](https://docs.substrate.io/reference/glossary/#frame) utilities, a functional effect system (Rune) and a fluent API, which facilitates multistep, multichain interactions without compromising either performance or safety.
+Capi is a framework for crafting interactions with Substrate chains. It consists
+of a development server and fluent API, which facilitates multistep, multichain
+interactions without compromising either performance or ease of use.
 
+- [Docs &rarr;](https://docs.capi.dev)<br />Guides for Capi developers and
+  pattern library developers
 - [Examples &rarr;](./examples)<br />SHOW ME THE CODE
-- [API Reference &rarr;](https://deno.land/x/capi/mod.ts)<br />A generated API reference, based on type signatures and in-source comments.
-- [Type Conversion Guide &rarr;](./docs/Types.md)<br />Guide for Capi's conversion of types from Rust to TypeScript
+- [API Reference &rarr;](https://deno.land/x/capi/mod.ts)<br />A generated API
+  reference, based on type signatures and TSDocs.
 
 ## At a Glance
 
-Run the local server.
+### Node.js Usage
+
+Run the Capi server.
 
 ```sh
-deno run -A https://deno.land/x/capi/main.ts
+npx capi
 ```
 
-Then, open your IDE and import pallet-corresponding modules from the local server.
+NPM-install chain-specific APIs from the Capi server.
+
+`package.json`
+
+```json
+{
+  "dependencies": {
+    "polkadot": "http://localhost:4646/frame/wss/rpc.polkadot.io/@latest/pkg.tgz"
```

Comment:
This should be `@capi/polkadot` and should be from `capi.dev`, not the local server